sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -56,14 +56,14 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 69a56ea6d1369a6bde5fce97c85fc7dbda49252f
-  # Merge: b25bb64d b78f541d
+  set(UNIFIED_RUNTIME_REPO "https://github.com/PietroGhg/unified-runtime.git")
+  # commit 659d3f469faa99a886fa680a3d6d20449b109578
+  # Merge: 192e9404 f94550b4
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Mon Dec 11 12:30:24 2023 +0000
-  #     Merge pull request #1123 from aarongreig/aaron/usmLocationProps
-  #     [OpenCL] Add ur_usm_alloc_location_desc struct and handle it in the CL adapter.
-  set(UNIFIED_RUNTIME_TAG 69a56ea6d1369a6bde5fce97c85fc7dbda49252f)
+  # Date:   Tue Nov 14 16:45:24 2023 +0000
+  #     Merge pull request #1059 from martygrant/martin/moveNativeCPUAdapterToUR
+  #     [NATIVECPU] Move Native CPU adapter to UR.
+  set(UNIFIED_RUNTIME_TAG pietro/barriers)
```

https://github.com/oneapi-src/unified-runtime/pull/1152 has now merged so please pull in the latest sycl branch changes, resolve the conflicts, update the UR tag, then make this ready for review. 

```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit eddda12352c27961fb454331f88e05d5335db7ed
  # Merge: 6892412f 71e7e38e
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Wed Jan 24 12:59:32 2024 +0000
  #     Merge pull request #1152 from PietroGhg/pietro/barriers
  #     [NATIVECPU] Handle oneAPI Construction Kit integration in runtime
  set(UNIFIED_RUNTIME_TAG eddda12352c27961fb454331f88e05d5335db7ed)
```